packages/ssz/test/perf/proxy.test.ts

```diff
@@ -16,6 +16,7 @@ describe("Access object properties in generated objects", () => {
 
   class CustomObj {}
   for (const key in rawObject) {
+    if (!Object.hasOwn(rawObject, key)) continue;
```

Comment:
This feels like a code change.  Why is this necessary? Should this be a thrown error instead of `continue`?  This deserved broader discussion though.

Reply:
So when you do `in` loop the JS loops over the prototype properties as well, which is not usually not the intention of developer and so why can cause bugs. So why this rule requires to add a condition on property check. 

https://biomejs.dev/linter/rules/use-guard-for-in

Reply:
ahhh!! ok.  I added a comment for future maintainers